core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -369,11 +416,12 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
   def updateSubmissionStatus(dataAccess: DataAccess)(implicit executionContext: ExecutionContext): ReadWriteAction[Boolean] = {
     dataAccess.workflowQuery.listWorkflowRecsForSubmissionAndStatuses(submissionId, (WorkflowStatuses.queuedStatuses ++ WorkflowStatuses.runningStatuses):_*) flatMap { workflowRecs =>
       if (workflowRecs.isEmpty) {
-        dataAccess.submissionQuery.findById(submissionId).map(_.status).result.head.map { status =>
-          SubmissionStatuses.withName(status) match {
+        dataAccess.submissionQuery.findById(submissionId).map(_.status).result.head.flatMap { status =>
+          val finalStatus = SubmissionStatuses.withName(status) match {
             case SubmissionStatuses.Aborting => SubmissionStatuses.Aborted
             case _ => SubmissionStatuses.Done
           }
+          if(config.enableEmailNotifications) sendTerminalSubmissionNotification(submissionId, finalStatus).map(_ => finalStatus) else DBIO.successful(finalStatus)
```

Comment:
This is behind a config setting now because we want to roll the code out so it doesn't get stale, but we need to overhaul the Terra UI profile page to allow for users to turn this notification off/on.